cli/templateedit.go

```diff
@@ -32,6 +33,7 @@ func (r *RootCmd) templateEdit() *clibase.Cmd {
 		allowUserAutostart             bool
 		allowUserAutostop              bool
 		requireActiveVersion           bool
+		deprecatedMessage              string
```

Comment:
```suggestion
		deprecationMessage             string
```
Thoughts on renaming?